modules/workflow-workflowoperation/src/main/java/org/opencastproject/workflow/handler/workflow/StartWorkflowWorkflowOperationHandler.java

```diff
@@ -51,7 +53,7 @@
   private static final Logger logger = LoggerFactory.getLogger(StartWorkflowWorkflowOperationHandler.class);
 
   /** Name of the configuration option that provides the media package ID */
-  public static final String MEDIA_PACKAGE_ID = "media-package";
+  public static final String MEDIA_PACKAGE_ID = "media-packages";
```

Comment:
This is a breaking change that is not allowed in OC 11. I would suggest only deprecating the `media-package` option but still allowing it. It should be possible to simply merge the `media-packages` and `media-package` strings possible with a seperator.

Reply:
should be fixed